### ISSUE TITLE
hungryroot.com: breakage

### ISSUE THREAD

**Issue by wfcoder** [uBlockOrigin/uAssets#18190]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.hungryroot.com/myhungryroot/`

### Category

breakage

### Description

Page doesn't load at all when logged into the site with ublock enabled in both Chrome and Firefox. White screen only.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>
![Wdxe2C6](https://github.com/uBlockOrigin/uAssets/assets/8601671/0d67d60a-f6a5-48d9-bc11-00c31787e9bf)

### Configuration

<details>

```yaml
uBlock Origin: 1.49.2
Firefox: 113
filterset (summary): 
 network: 92317
 cosmetic: 38748
 scriptlet: 17443
 html: 942
listset (total-discarded, last-updated): 
 default: 
  user-filters: 0-0, never
  ublock-filters: 33909-73, 2d.3h.51m
  ublock-badware: 7117-0, 2d.3h.51m
  ublock-privacy: 442-0, 2d.3h.51m
  ublock-abuse: 70-70, 2d.3h.51m
  ublock-unbreak: 2031-1, 2d.3h.51m
  ublock-quick-fixes: 352-0, 2h.41m
  easylist: 65871-667, 2d.3h.51m
  easyprivacy: 31844-615, 2d.3h.50m
  urlhaus-1: 5659-0, 40m
  plowe-0: 3735-4, 2d.3h.50m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
 allReadyAfter: 149 ms (selfie)
 maxAssetCacheWait: 47 ms
popupPanel: 
 blocked: 9
 network: 
  datadoghq-browser-agent.com: 4
  facebook.net: 1
  google-analytics.com: 2
  googletagmanager.com: 1
  segment.com: 1
```
</details>

**Comment by mapx-** [comment]
- open the uBO's logger: https://github.com/gorhill/uBlock/wiki/The-logger
- repeat the steps to get the issue
- go back into the uBO's logger and then copy and paste here all blocked requests

if you want to provide (in private via email) the credentials of your account let us know

**Comment by mapx-** [comment]
and just to be sure it's all ok with your uBO / lists, follow all these indications:

- clear cookies / localStorage/ cache for the tested page
- reset uBO, don't modify any uBO setting, update all the filter lists, don't add any other list (other than your regional list)
- disable any custom filter for that site
- disable any other extension / addon in your browser, disable builtin blockers /shields/ tracking protection in brave/ opera/ vivaldi/ edge/ firefox etc
- if Opera browser: open opera://extensions/ and check the "allow access to search page results" option
- disable (if any) the web shields in your antivirus / antimalware software / other blocking software
- are you using a DNS which features adblocking (like AdGuard DNS)?=> test using another DNS (google for example =>  IP addresses 8.8.8.8 & 8.8.4.4)
- restart the system, reopen the browser, test again.

**Comment by wfcoder** [comment]
<details><summary>Logger output</summary>

```
+1
www.hungryroot.com
3
get
image
https://d1vd0kmlu4o34z.cloudfront.net/public/img/favicons/favicon-192.png
```
```
+1
www.hungryroot.com
3
get
image
https://d1vd0kmlu4o34z.cloudfront.net/public/img/favicons/favicon-192.png
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/v2/customers/4834997/experiments/
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/v2/customers/4834997/pricing_config
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/v2/experiments/?limit=500
```
```
+0
google-analytics_analytics.js
<<
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
||google-analytics.com/analytics.js$script,redirect-rule=google-analytics_analytics.js:5
--
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
||google-analytics.com^
--
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/get_customer/?includes=all
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/get_customer_dashboard/?future_orders_count=3&past_orders_count=2
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/get_customer_dashboard/?future_orders_count=1&past_orders_count=0
```
```
+0
www.hungryroot.com
1
get
xhr
https://www.hungryroot.com/api/v2/customers/self
```
```
+0
google-analytics_analytics.js
<<
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
||google-analytics.com/analytics.js$script,redirect-rule=google-analytics_analytics.js:5
--
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
||google-analytics.com^
--
www.hungryroot.com
3
get
script
https://www.google-analytics.com/analytics.js
```
```
+0
||datadoghq-browser-agent.com^$3p
--
www.hungryroot.com
3
get
script
https://www.datadoghq-browser-agent.com/us1/v4/datadog-rum.js
```
```
+0
www.hungryroot.com
3
get
script
https://apis.google.com/_/scs/abc-static/_/js/k=gapi.lb.en.yTSbWrSe458.O/m=client/rt=j/sv=1/d=1/ed=1/rs=AHpOoo_5dpdVCMe_LPx1yH-hHA2M85TB-Q/cb=gapi.loaded_0?le=scs
```
```
+0
||datadoghq-browser-agent.com^$3p
--
www.hungryroot.com
3
get
script
https://www.datadoghq-browser-agent.com/datadog-logs-us.js
```
```
+0
googletagmanager_gtm.js
<<
www.hungryroot.com
3
get
script
https://www.googletagmanager.com/gtm.js?id=GTM-KJTW7JH&gtm_auth=4IhxdwkkE9Co-amw3i5jLQ&gtm_preview=env-2&gtm_cookies_win=x
```
```
+0
||googletagmanager.com/gtm.js$script,redirect-rule=googletagmanager_gtm.js:5
--
www.hungryroot.com
3
get
script
https://www.googletagmanager.com/gtm.js?id=GTM-KJTW7JH&gtm_auth=4IhxdwkkE9Co-amw3i5jLQ&gtm_preview=env-2&gtm_cookies_win=x
```
```
+0
||googletagmanager.com^
--
www.hungryroot.com
3
get
script
https://www.googletagmanager.com/gtm.js?id=GTM-KJTW7JH&gtm_auth=4IhxdwkkE9Co-amw3i5jLQ&gtm_preview=env-2&gtm_cookies_win=x
```
```
+0
||cdn.segment.com^
--
www.hungryroot.com
3
get
script
https://cdn.segment.com/analytics.js/v1/CyQrJwUY84tTss8ySJdBZJydsqTm76W3/analytics.min.js
```
```
+0
/fbevents.js
--
www.hungryroot.com
3
get
script
https://connect.facebook.net/en_US/fbevents.js
```
```
+0
www.hungryroot.com
3
get
script
https://d1vd0kmlu4o34z.cloudfront.net/public/js/react/spa.js?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
script
https://d1vd0kmlu4o34z.cloudfront.net/public/js/utils/facebook_signup_helper.js?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
script
https://d1vd0kmlu4o34z.cloudfront.net/public/js/utils/api_signup_helper.js?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
script
https://d1vd0kmlu4o34z.cloudfront.net/public/js/base_v3.js?v=5123cad8
```
```
+0
||datadoghq-browser-agent.com^$3p
--
www.hungryroot.com
3
get
script
https://www.datadoghq-browser-agent.com/us1/v4/datadog-rum.js
```
```
+0
www.hungryroot.com
3
get
script
https://cdnjs.cloudflare.com/ajax/libs/jquery.easytabs/3.2.0/jquery.easytabs.min.js
```
```
+0
www.hungryroot.com
3
get
script
https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js
```
```
+0
www.hungryroot.com
3
get
script
https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js
```
```
+0
www.hungryroot.com
3
get
script
https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.0/jquery-ui.min.js
```
```
+0
www.hungryroot.com
3
get
script
https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js
```
```
+0
www.hungryroot.com
3
get
script
https://apis.google.com/js/api:client.js
```
```
+0
www.hungryroot.com
3
get
script
https://apis.google.com/js/platform.js
```
```
+0
||datadoghq-browser-agent.com^$3p
--
www.hungryroot.com
3
get
script
https://www.datadoghq-browser-agent.com/datadog-logs-us.js
```
```
+0
www.hungryroot.com
3
get
css
https://d1vd0kmlu4o34z.cloudfront.net/public/css/spa.css?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
css
https://d1vd0kmlu4o34z.cloudfront.net/public/css/edit_delivery.css?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
css
https://d1vd0kmlu4o34z.cloudfront.net/public/css/react_components.css?v=5123cad8
```
```
+0
www.hungryroot.com
3
get
css
https://d1vd0kmlu4o34z.cloudfront.net/public/css/base_v3.css?v=5123cad8
```
```
+0
www.hungryroot.com
1
get
doc
https://www.hungryroot.com/myhungryroot/
```
```
+0
https://www.hungryroot.com/myhungryroot/
```
</details>

**Comment by mapx-** [comment]
**Test 1 by 1 these filters:**

```
@@||datadoghq-browser-agent.com/datadog-$script,domain=hungryroot.com
@@||cdn.segment.com/analytics.js/$script,domain=hungryroot.com
@@||googletagmanager.com/gtm.js$script,domain=hungryroot.com
```

if still broken try combinations of 2, 3

**Comment by wfcoder** [comment]
Tried each 1 by 1 as well as combos of 2 and all 3 filters to no avail, assuming I'm applying them correctly?

![image](https://github.com/uBlockOrigin/uAssets/assets/8601671/6fa76ca4-435a-4880-bb3e-f94a4410e2f2)

**Comment by wfcoder** [comment]
Just some extra info - disabling uBlock on the page works, but there's some type of element on top of the entire page that is being blocked, which I imagine is causing the issue.

![image](https://github.com/uBlockOrigin/uAssets/assets/8601671/8e4d5684-d0db-4b8f-bf6b-3a694b7c30e2)

**Comment by wfcoder** [comment]
Tested the new filter, still no dice. And I did follow the indications listed. I have no custom filters on uBlock, i just run everything default out of the box. No antivirus/firewall installed, just Windows Defender and no other extensions running in Firefox.

**Comment by wfcoder** [comment]
Yes that is fine. Just don't change my food menu!

**Comment by mapx-** [comment]
send the info here ( `axis.trivia@gmail.com` ) + eventually steps to reproduce

**Comment by wfcoder** [comment]
Sent.

**Comment by mapx-** [comment]
This filter seems working fine on my end:

`@@||datadoghq-browser-agent.com^$script,domain=hungryroot.com`

Test on your side too

**Comment by wfcoder** [comment]
Yep, that seems to have done it. Thanks!
